plugins/cassandra/CMakeLists.txt

```diff
@@ -29,12 +29,21 @@ project(cassandraembed)
 # until then, we build the required libraries from source
 # Build libuv, required by the cassandra driver but not available on all distros
 if(USE_CASSANDRA)
-  add_custom_command(
-    OUTPUT ${PROJECT_BINARY_DIR}/.libs/libuv${CMAKE_SHARED_LIBRARY_SUFFIX}
-    COMMAND sh ${PROJECT_SOURCE_DIR}/libuv/autogen.sh
-    COMMAND ${PROJECT_SOURCE_DIR}/libuv/configure
-    COMMAND make
-    WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
+  if(APPLE)
+	add_custom_command(
+	  OUTPUT ${PROJECT_BINARY_DIR}/.libs/libuv${CMAKE_SHARED_LIBRARY_SUFFIX}
+	  COMMAND sh ${PROJECT_SOURCE_DIR}/libuv/autogen.sh
+	  COMMAND ${PROJECT_SOURCE_DIR}/libuv/configure --libdir=${CMAKE_INSTALL_PREFIX}/${LIB_DIR}
```

Comment:
Not sure why this would only be true for Apple builds?

Reply:
The build for libuv does include basic -rpath support, but macOS doesn't support rpath the same way most other Linux systems do.  It appears that libuv simply doesn't take macOS into account, so the rpath is broken in this case.

This PR links libuv with a full path rather than through @rpath (visible if you use otool).